pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2197,20 +2200,31 @@ func checkIoBundle(ctx *domainContext, ib *types.IoBundle) error {
 	return nil
 }
 
+// Move the USB controllers to/from pciback based on usbAccess
+// Also enable/disable usbhid and related mouse/keyboard based on that
+// XXX should we have a separate knob for HID and for usb-storage?
 func updateUsbAccess(ctx *domainContext) {
 
 	log.Infof("updateUsbAccess(%t)", ctx.usbAccess)
 	if !ctx.usbAccess {
-		maybeAssignableAdd(ctx)
+		if removeUSBfromKernel() {
+			maybeUSBAssignableAdd(ctx)
+		}
 	} else {
-		maybeAssignableRem(ctx)
+		if maybeUSBAssignableRem(ctx) {
+			addUSBtoKernel()
+		}
 	}
 	checkIoBundleAll(ctx)
 }
 
-func maybeAssignableAdd(ctx *domainContext) {
+// Try to add all of USB group to pciback
+// Returns success/failure
+func maybeUSBAssignableAdd(ctx *domainContext) bool {
```

Comment:
I am confused about this function rename -- why are you changing the name when literally nothing in the function's body changing @eriknordmark ? I mean I could see if you pull the add/removeUSBtoKernel into the function body, but like it is -- I'm confused.

Reply:
The old function name was confusing; the code hasn't been doing anything but USB assignment for a year or two.